velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -248,6 +272,7 @@ struct UnixTimestampParseWithFormatFunction
  private:
   bool isConstFormat_{false};
   bool invalidFormat_{false};
+  bool legacyDateFormatter_{false};
```

Comment:
nit: legacyDateFormatter_ -> legacyFormatter_ since Date is implied

Reply:
Thanks. Done.